conf/application.conf

```
@@ -241,6 +241,14 @@ datastore {
     # The credentials are selected by uri prefix, so different s3 uri styles may need duplicated credential entries.
     credentials = []
   }
+  s3Upload {
+    enabled = false
+    # Use the name (prefix) of a credential in the dataVaults section here to use it for uploads.
+    credentialName = "my-credential"
+    endpoint = "https://custom-s3-endpoint.example.local"
+    bucketName = "your-bucket-name"
+    objectKeyPrefix = "webknossos-uploads"
```

The credential name already contains the endpoint url, bucket name and optionally a prefix. Can we reuse this here?

```suggestion
    credentialName = "s3://example/uri/prefix"
    objectKeyPrefix = "webknossos-uploads" # optional additional prefix
```